src/fetch-page-data.js

```diff
@@ -12,33 +12,31 @@ chrome.runtime.sendMessage({
   abTestBuckets: getAbTestBuckets(),
   windowHeight: window.innerHeight,
   highlightState: window.highlightComponent.state
-});
+})
 
-function getMetatag(name) {
+function getMetatag (name) {
   var meta = document.getElementsByTagName('meta')[name]
   return meta && meta.getAttribute('content')
 }
 
-function getAbTestBuckets() {
-  var abMetaTags = document.querySelectorAll('meta[name="govuk:ab-test"]');
+function getAbTestBuckets () {
+  var abMetaTags = document.querySelectorAll('meta[name="govuk:ab-test"]')
 
-  var metaTagPattern = /([\w-]+):([\w-]+)/;
-  var buckets = {};
+  var metaTagPattern = /([\w-]+):([\w-]+)/
+  var buckets = {}
 
   abMetaTags.forEach(function (metaTag) {
-    var testNameAndBucket = metaTagPattern.exec(metaTag.content);
-    var testName = testNameAndBucket[1];
-    var currentBucket = testNameAndBucket[2];
+    var testNameAndBucket = metaTagPattern.exec(metaTag.content)
+    var testName = testNameAndBucket[1]
+    var currentBucket = testNameAndBucket[2]
     var allowedBuckets =
-      (metaTag.dataset.allowedVariants || "A,B").split(",");
+      (metaTag.dataset.allowedVariants || 'A,B').split(',')
 
     buckets[testName] = {
       currentBucket: currentBucket,
       allowedBuckets: allowedBuckets
     }
-  });
+  })
 
-  return buckets;
+  return buckets
 }
-
-undefined;
```

Comment:
I think we should remove `undefined` from commit 4b2f6f1bd152082f72f8245a0c7ed4eaec81a257 (which will then remove the need to undo the change in commit 13db43a74d2444f17f97ef6c3a05d531c9e87aa8).

Reply:
Yeah, I agree, I also find the manual inclusion of the `jasmine-jquery` library here unusual. There is a similar approach in the publishing components gem, [modernizr.js](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/vendor/modernizr.js) for example, but these are stored in a `vendor` directory, which I think makes it clear that they are external libraries, as opposed to the directory of `helpers` used here, which could just be used for internal helpers we have created.

Would be good to review this again in the future to improve things, possibly by one or a mix of the approaches below:

- Move the libraries to a vendor directory
- Switch to using `node_modules`
- See if any of the libraries can be removed